transports/uds/Cargo.toml

```diff
@@ -18,7 +18,7 @@ tokio = { workspace = true, default-features = false, features = ["net"], option
 tracing = { workspace = true }
 
 [dev-dependencies]
-tempfile = "3.10"
+tempfile = "3.6"
```

Comment:
Are we okay with this downgrade? I think it should be fine since it's a dev-dependency.